src/main/kotlin/no/nav/syfo/Environment.kt

```diff
@@ -39,7 +39,8 @@ data class Environment(
     val electorPath: String = getEnvVar("ELECTOR_PATH"),
 
     val outdatedDialogmotesvarCutoff: LocalDate = LocalDate.parse(getEnvVar("OUTDATED_DIALOGMOTESVAR_CUTOFF")),
-    val kakfaConsumerAktivitetskravExpiredVarselEnabled: Boolean = getEnvVar("TOGGLE_KAFKA_CONSUMER_AKTIVITETSKRAV_EXPIRED_VARSEL").toBoolean()
+    val kakfaConsumerAktivitetskravExpiredVarselEnabled: Boolean = getEnvVar("TOGGLE_KAFKA_CONSUMER_AKTIVITETSKRAV_EXPIRED_VARSEL").toBoolean(),
+    val isAktivitetskravVurderingConsumerEnabled: Boolean = getEnvVar("IS_AKTIVITETSKRAV_VURDERING_CONSUMER_ENABLED").toBoolean(),
```

Comment:
Trenger kanskje ikke `is` foran? Bør i hvert fall bestemme oss for en av delene :sweat_smile:

Reply:
Tenkte bare siden det er formatet vi har på unleash-togglene våre 🤷🏼‍♂️